apps/sms/js/thread_list_ui.js

```diff
@@ -530,7 +490,64 @@ var ThreadListUI = {
         text: 'delete',
         className: 'danger'
       }
-    ).then(performDeletion.bind(this));
+    ).then(performDeletion.bind(this, selected));
```

Comment:
we don't really need performDeletion if we remove the loop, we can move what's remaining of performDeletion to `deleteThreadDraftUI` directly and call it directly here. Wha